### PR TITLE
New package: parsec-147.9

### PR DESCRIPTION
**Pull request by Kozova1**

### CODE REVIEW

File: srcpkgs/parsec/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+archs="x86_64"
+build_style=fetch
+depends="wget"
+short_desc="Low latency proprietary game streaming application"
+maintainer="Kozova1 <mug66kk@gmail.com>"
+license="Proprietary"
+homepage="https://parsecgaming.com"
+distfiles="https://s3.amazonaws.com/parsec-build/package/parsec-linux.deb"
+checksum=beed951ebee0189e8c22aa3d9190e1ddc7691c4703379e146a8b812c21b3a273
+nopie=" "
```
Comment (maxice8) [inline srcpkgs/parsec/template]: nopie=yes

File: srcpkgs/parsec/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+do_extract() {
+	ar x "${XBPS_BUILDDIR}/parsec-linux.deb"
+}
+
+do_install() {
+	tar xf "data.tar.xz" -C ${DESTDIR}
+	vmkdir usr/lib/parsec
+	mv ${DESTDIR}/usr/share/parsec/skel/parsecd-147-9.so ${DESTDIR}/usr/lib/parsec/parsecd-147-9.so
+	ln -s ${DESTDIR}/usr/lib/parsec/parsecd-147-9.so ${DESTDIR}/usr/share/parsec/skel/
```
Comment (maxice8) [inline srcpkgs/parsec/template]: ln -rs can be used

**Comment by Kozova1** [comment]
terribly sorry for the multiple commits

**Comment by Kozova1** [comment]
What happened here? Should I add `ar` as a dependency?

File: srcpkgs/parsec/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+short_desc="Low latency proprietary game streaming application"
+maintainer="Kozova1 <mug66kk@gmail.com>"
+license="Proprietary"
+homepage="https://parsecgaming.com"
+distfiles="https://s3.amazonaws.com/parsec-build/package/parsec-linux.deb"
+checksum=beed951ebee0189e8c22aa3d9190e1ddc7691c4703379e146a8b812c21b3a273
+nopie=yes
+
+do_extract() {
+	ar x "${XBPS_BUILDDIR}/parsec-linux.deb"
```
Comment (maxice8) [inline srcpkgs/parsec/template]: use ${XBPS_SRCDISTDIR}/${pkgname}-${veersion} instead

**Comment by Kozova1** [comment]
gotcha

**Comment by Kozova1** [comment]
umm build passed, what now?

**Comment by Kozova1** [comment]
Added ufetch package - no idea how to separate pull requests

**Comment by maxice8** [comment]
On Mon, Mar 25, 2019 at 10:34:42AM -0700, Kozova1 wrote:
> Added ufetch package - no idea how to separate pull requests
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10232#issuecomment-476302760

Please don't, use a different branch

**Comment by Kozova1** [comment]
This seems better

**Comment by Kozova1** [comment]
just wondering what's going on, github shows "1 review requesting changes" but it was already modified

**Comment by silvernode** [comment]
I am currently using parsec by installing all deps manually and running a binary. A package would be very nice indeed so I show my support.

**Comment by Kozova1** [comment]
This fail seems to be a false positive, it times out in bootstrapping.

**Comment by silvernode** [comment]
I got this issue subscribed for notifications. Hoping I can use xbps to install parsec at some point in the future. Let me know if you need help with testing or anything else.

**Comment by silvernode** [comment]
Oh cool, it looks like it finally passed travis.

**Comment by maxice8** [comment]
Parsec seems to only link to these libraries.

```
   SONAME: libm.so.6 <-> glibc>=2.29_1
   SONAME: libpthread.so.0 <-> glibc>=2.29_1
   SONAME: libc.so.6 <-> glibc>=2.29_1
   SONAME: libdl.so.2 <-> glibc>=2.29_1
   SONAME: libstdc++.so.6 <-> libstdc++>=4.4.0_1
   SONAME: libgcc_s.so.1 <-> libgcc>=4.4.0_1
   SONAME: libGL.so.1 <-> libGL>=7.11_1
   SONAME: libX11.so.6 <-> libX11>=1.2_1
```

Are the others in depends= really required ?

**Comment by silvernode** [comment]
> Parsec seems to only link to these libraries.
> 
> ```
>    SONAME: libm.so.6 <-> glibc>=2.29_1
>    SONAME: libpthread.so.0 <-> glibc>=2.29_1
>    SONAME: libc.so.6 <-> glibc>=2.29_1
>    SONAME: libdl.so.2 <-> glibc>=2.29_1
>    SONAME: libstdc++.so.6 <-> libstdc++>=4.4.0_1
>    SONAME: libgcc_s.so.1 <-> libgcc>=4.4.0_1
>    SONAME: libGL.so.1 <-> libGL>=7.11_1
>    SONAME: libX11.so.6 <-> libX11>=1.2_1
> ```
> 
> Are the others in depends= really required ?
My guess is either those are the core libraries for the streaming portion and that the freetype fonts package (for example) is possibly used in some other way not shown. 

With that said, a better bet is that the support page with the list of dependencies is outdated since Parsec no longer requires the use of it's stand-alone desktop app. These days the user runs ```app_daemon=1``` which triggers the default browser and uses that for all the interface rendering. 

I recommend that @Kozova1 makes a local change to the package with only the dependencies linked against and see if the parsec daemon functions properly.

**Comment by Kozova1** [comment]
got official statement from devs, pango freetype and cairo are not needed

**Comment by maxice8** [comment]
Merged manually (will make it seem like this PR was closed)

**Comment by silvernode** [comment]
Well that's exciting, we have parsec in a package now. No more manual fiddling on fresh installs. Updating will also be easier hopefully.

**Pull request closed without merging**